chromium/browser_type.go

```diff
@@ -60,34 +60,34 @@ func NewBrowserType(vu k6modules.VU) api.BrowserType {
 
 func (b *BrowserType) init(
 	opts goja.Value, isRemoteBrowser bool,
-) (context.Context, *common.LaunchOptions, *log.Logger, error) {
+) (context.Context, *common.BrowserOptions, *log.Logger, error) {
```

Comment:
Not so sure about this one because for me `launch` seems to indicate that we are actually always launching a process. I see `BrowserOptions` as the options for the browser itself. In this case I would prefer to keep it like this :sweat_smile: